openapi-python-templates/requirements.mustache

```diff
@@ -0,0 +1,4 @@
+pydantic >= 1.0
+httpx >= 0.10
+fastapi >= 0.40
```

Comment:
Initially this was set to 0.42.0, so I wouldn't go lower than that unless you want to do the extra check. Also, I wish if we would get rid of this dependency...